transformer_engine/pytorch/csrc/extensions.cu

```diff
@@ -1812,6 +1980,8 @@ PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
                   "Fused Attention FP8/BF16/FP16 BWD with packed KV");
   m.def("fp8_transpose", &fp8_transpose, "Transpose with FP8 I/O");
   m.def("fp8_gelu", &fp8_gelu, "GeLU with FP8 output");
+  m.def("fa_prepare", &fa_prepare, "Prepare QKV for Flash Attention");
```

Comment:
appending a `_fwd` would be consistent with naming of other similar extensions

Reply:
Ok, will add that.